worker/worker.go

```diff
@@ -294,8 +295,10 @@ type worker struct {
 	pool      *sessionPool
 	masterKey [32]byte
 
-	accounts    *accounts
-	priceTables *priceTables
+	accounts *accounts
+
+	priceTablesMu sync.Mutex
+	priceTables   map[types.PublicKey]*priceTable
```

Comment:
Is there any specific reason for inlining this? I kinda appreciated the smaller scope of methods on the worker. Felt very nice to have all pricetable code isolated in its own type.

Reply:
It used to be a `map` a `mutex` and something else. We removed the "something else" so I figured it made less sense now to have it be its own type. I reverted it though I don't mind.